docs/toolchains/version-records/2026-08-20-esp32-flash-boot-native.md

```
@@ -220,3 +226,155 @@ I (364) main_task: Returned from app_main()
+
+`Commands run` は上の「再現できる形」と同じである。**追加した command は無い。**
+
+### size report
+
+```text
+debug   ELF 14,214,220 bytes / flash image 574,512 bytes
+release ELF    672,160 bytes / flash image 390,928 bytes
+```
+
+`espflash` の表示は `App/part. size: 390,928/4,128,768 bytes, 9.47%` である。
+#6 の release image 381,344 bytes から **9,584 bytes 増えた**。partition に対する比率は 9.24 % → 9.47 %。
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)